.github/ISSUE_TEMPLATE/vote.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Vote
+about: Vote to make a decision related to an RFC 
+title: '[Vote] RFC #XXXX:'
+labels: "rfc:vote-needed"
+assignees: ''
+---
+
+A [vote](https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/rfcs/README.md#voting) has been called for RFC #XXXX following the RFC process.
+
+### Stakeholders
+
+Any person in the community may vote. Votes of the stakeholders are binding. The stakeholders as defined in the RFC are:
+
+| Stakeholders |
+|-------------|
+| @open-telemetry/collector-approvers |
```

Comment:
It may be helpful for onlookers if we explicitly state the current number of stakeholders and the number of votes required to pass.

Related: There's an edge case where membership in the approvers group changes while the vote is open. Should we lock these numbers down when the vote is opened, or allow them to change until the vote is closed?

Reply:
> Should we lock these numbers down when the vote is opened, or allow them to change until the vote is closed?

This is interesting. I added a sentence noting that we lock these numbers down when the vote starts, since I feel like this is the more practical option, but if you feel differently please let me know, I am undecided as to what option is best